core-processor/src/processing.rs

```diff
@@ -222,44 +223,6 @@ fn process_error(
         amount: gas_burned,
     });
 
-    // We check if value is greater than zero to don't provide
```

Comment:
why was it moved?

Reply:
New error case makes early return so it doesn't need these notes